packages/mobile/src/qrcode/QRScanner.tsx

```diff
@@ -85,7 +88,7 @@ class QRScanner extends React.Component<Props> {
               }}
               style={styles.preview}
               type={RNCamera.Constants.Type.back}
-              onBarCodeRead={memoize(this.onBarCodeDetected, (qr) => qr.data)}
```

Comment:
The way it was before, we were creating a new memoize function on each render, defeating the purpose of memoization I believe.

Reply:
interesting -- nice catch